python/lsst/verify/bin/print_metricvalues.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of verify.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Summarize measured metric values in one butler repo, or difference values
+between two repos.
+"""
+__all__ = ["main", "build_argparser"]
+
+import argparse
+
+import lsst.daf.butler
+from .. import extract_metricvalues
+
+
+def build_argparser():
+    """Return an ArgumentParser for this script.
+    """
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog='More information is available at https://pipelines.lsst.io.')
```

Comment:
That comes from the example template: https://developer.lsst.io/stack/argparse-script-topic-type.html#the-autoprogram-directive

Reply:
If it's just there as example fluff, I'd say leave it out...

Reply:
I've asked about this on #dm-docs-support on slack.

Reply:
The word from Jonathan is: "My intent with that advice was to be a helpful way of connecting a random executable that someone might find with the larger Science Pipelines project so that could understand where it came from."